File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        // Prompt user for height until the user inputs a valid number.
        height = get_int("Height : ");
    }
    while (height < 1 || height > 8);

    // Print the pyramids
    for (int i = 0; i < height; i++)
    {
        // Space
        for (int j = height - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        // Hash
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        // the middle
        printf("  ");

        // Hash
        for (int n = 0; n <= i; n++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


# make the # as requested
def main():
    height = get_height()
    for i in range(height):
        for j in range(height - i - 1):
            print(" ", end="")

        for k in range(i + 1):
            print("#", end="")

        print("  ", end="")

        for l in range(i + 1):
            print("#", end="")
        print("")


# get height from user
def get_height():
    while True:
        n = get_int("Height: ")
        if 0 < n < 9:
            return n


main()